cypress/e2e/data-import/field-protection-overrides.cy.js

```diff
@@ -5,7 +5,7 @@ import SettingsMenu from '../../support/fragments/settingsMenu';
 import NewFieldMappingProfile from '../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
 import FieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
 import MarcFieldProtection from '../../support/fragments/settings/dataImport/marcFieldProtection';
-import MappingProfileDetails from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetails';
+import MappingProfileDetailsView from '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView';
```

Comment:
Unable to resolve path to module '../../support/fragments/data_import/mapping_profiles/mappingProfileDetailsView'
I think it should be '../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView'. Check please

Reply:
Done